Dockerfile

```diff
@@ -33,39 +31,55 @@ ARG goproxy=https://proxy.golang.org
 # Run this with docker build --build-arg package=./controlplane/kubeadm or --build-arg package=./bootstrap/kubeadm
 ENV GOPROXY=$goproxy
 
+RUN yum update -y \
+    && yum-config-manager --save --setopt=ol7_ociyum_config.skip_if_unavailable=true \
+    && yum install -y oracle-golang-release-el7 \
+    && yum-config-manager --disable ol7_developer_golang\* \
+    && yum-config-manager --enable ol7_developer_golang119 \
+    && yum -y install golang \
+    && yum clean all \
+    && go version
+
+RUN go env GOPATH
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
 
 # Cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-RUN --mount=type=cache,target=/go/pkg/mod \
-    go mod download
+RUN go mod download
 
 # Copy the sources
 COPY ./ ./
 
-# Cache the go build into the Go’s compiler cache folder so we take benefits of compiler caching across docker build calls
-RUN --mount=type=cache,target=/root/.cache/go-build \
-    --mount=type=cache,target=/go/pkg/mod \
-    go build .
+## Cache the go build into the Go’s compiler cache folder so we take benefits of compiler caching across docker build calls
```

Comment:
remove  comments

Reply:
It was not intended. Thanks for finding it. Will fix it.

Reply:
Added go build. Thanks.